scrapy/__init__.py

```diff
@@ -30,13 +26,22 @@
 # Scrapy and Twisted versions
 __version__ = (pkgutil.get_data(__package__, "VERSION") or b"").decode("ascii").strip()
 version_info = tuple(int(v) if v.isdigit() else v for v in __version__.split("."))
-twisted_version = (_txv.major, _txv.minor, _txv.micro)
 
 
-# Ignore noisy twisted deprecation warnings
-warnings.filterwarnings("ignore", category=DeprecationWarning, module="twisted")
```

Comment:
For some reason if I add this piece I get duplicated deprecation warning message, also not sure where should I place a test for this change.

Reply:
> if I add this piece I get duplicate deprecation warning message

Can you show an example?

I think this line shouldn't be removed, though it's probably not that useful, as IIRC `DeprecationWarning` warnings aren't printed anywhere by default.

Reply:
Sure, add the removed line after the warnings import and then call this snippet:

```python
from scrapy import twisted_version
print(twisted_version)
```

Reply:
Now i'm trying to figure out why tests are failing after I added ``__getattr__`` method. 😅

Reply:
Got it, fixed.